spec.md

```diff
@@ -738,8 +744,8 @@ This endpoint MAY be used for authentication/authorization purposes, but this is
 | end-5   | `PATCH`        | `/v2/<name>/blobs/uploads/<reference>`                         | `202`       | `404`/`416`       |
 | end-6   | `PUT`          | `/v2/<name>/blobs/uploads/<reference>?digest=<digest>`         | `201`       | `404`/`400`       |
 | end-7   | `PUT`          | `/v2/<name>/manifests/<reference>`                             | `201`       | `404`             |
-| end-8a  | `GET`          | `/v2/<name>/tags/list`                                         | `200`       | `404`             |
-| end-8b  | `GET`          | `/v2/<name>/tags/list?n=<integer>&last=<integer>`              | `200`       | `404`             |
+| end-8a  | `GET`          | `/v2/<name>/tags/list`                                         | `200`       | `404`/`400`       |
+| end-8b  | `GET`          | `/v2/<name>/tags/list?n=<integer>&last=<integer>`              | `200`       | `404`/`400`       |
```

Comment:
Just want to note I was confused when reading this section, because in a section above it says `&last=<tagname>`, and additionally "`<tagname>` MUST NOT be a numerical index, but rather it MUST be a proper tag".

Reply:
Rebased to include #464.